src/library_webgpu.js

```diff
@@ -390,20 +404,20 @@ var LibraryWebGPU = {
       'storage',
       'read-only-storage',
     ],
-    BufferMapState: [
-      'unmapped',
-      'pending',
-      'mapped',
-    ],
+    BufferMapState: {
```

Comment:
Quirk of the code generator, it's because `CompareFunction[0]` is expected to be used ever while `BufferMapState[0]` is not. I was thinking of smoothing this over in the generator along with other generation cleanups (like hopefully moving some generated code into separate files so we can get rid of "please copy-paste" steps)